lib/parse.ts

```diff
@@ -108,5 +119,5 @@ function generateSourceMap(
       })
     }
   })
-  return map.toJSON()
+  return JSON.parse(map.toString())
```

Comment:
Forgot to ask, why is this needed?

Reply:
It's not required with `0.6.1`. Please revert this line.

Reply:
😂I think I found the answer…
There was a`.d.ts` file added to the `source-map` package since v0.5.7, but missing `toJSON` declaration in `SourceMapGenerator` until v0.7.0. So we can't use this method unless we patch its type definitions…